planning/freespace_planner/src/freespace_planner/freespace_planner_node.cpp

```diff
@@ -191,6 +191,22 @@ Trajectory createStopTrajectory(const PoseStamped & current_pose)
   return createTrajectory(current_pose, waypoints, 0.0);
 }
 
+Trajectory createStopTrajectory(const Trajectory & trajectory)
```

Comment:
The old `Trajectory createStopTrajectory(const PoseStamped & current_pose)` can be removed because it is no longer used.

Reply:
It seems the old one is still used in line 448.

Reply:
if there is no available trajectory, use current pose to generate a stop trajectory. it seems redundant but  doesn't affect any performance.

Reply:
Oh, sorry, that's right. Is there any case where`partial_trajectory_` is empty?

Reply:
thanks, understood!